### PR TITLE
[qt5-3d] Support Visual Studio 2022

### PR DESCRIPTION
**Pull request by tansangxtt**
**Describe the pull request**
The error shows related to the qt5-3d module when I'm compiling COLMAP 3.6 by vcpkg on Windows 10 with Visual Studio 2022 installed. After tracing the logs, I realized that only assimp-vc143-mt.lib available in VCPKG_ROOT\installed\x64-windows\lib

- #### What does your PR fix?  
  N/A

- #### Which triplets are supported/not supported? Have you updated the [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt)?  
  No

- #### Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?  
  Yes

- #### If you have added/updated a port: Have you run `./vcpkg x-add-version --all` and committed the result?  
  Yes

**Host Environment**
OS: Windows 10
Compiler: Visual Studio 2022
**To Reproduce the error**
Steps to reproduce the behaviour:
vcpkg install qt5-3d:x64-windows

### CODE REVIEW

**Comment by JackBoosY** [comment]
@Neumann-A Does the qt upstream support Visual Studio 2022 now?

**Review by JackBoosY — APPROVED** [review]

File: versions/q-/qt5-3d.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "versions": [
     {
-      "git-tree": "f46ab15ed74e4680e16b61851b9eb41523a626f6",
+      "git-tree": "52bbd43162a0ad77ec4006cc2a999db2c03b6475",
```
Comment (JackBoosY) [inline versions/q-/qt5-3d.json:4]: Oh god, please revert this changes, bump the port version. See [documentation](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md#update-the-version-field-in-the-control-file-of-any-modified-ports).
Also please run command `vcpkg x-add-version --all` then commit the changes.

**Review by JackBoosY — CHANGES_REQUESTED** [review]

File: versions/q-/qt5-3d.json (hunk 1 above)
Comment (tansangxtt) [reply versions/q-/qt5-3d.json:4]: Apologise, I pushed the update

**Comment by tansangxtt** [comment]
@JackBoosY  I'm so sorry about the versioning; this is my first time here. Thank you!

**Comment by Neumann-A** [comment]
> Does the qt upstream support Visual Studio 2022 now?

no but that is not the question here.

Does assimp install pc files which could be used instead of hardcoding the library names?

**Review by JackBoosY — APPROVED** [review]

**Comment by JackBoosY** [comment]
> > Does the qt upstream support Visual Studio 2022 now?
> 
> no but that is not the question here.
> 
> Does assimp install pc files which could be used instead of hardcoding the library names?

Yes assimp has.

**Comment by JackBoosY** [comment]
LGTM, thanks for this PR!
